### ISSUE TITLE
Saving a CanvasRenderTarget to file turns opaque image areas gray

### ISSUE THREAD

**Issue by kellylawson** [Microsoft/Win2D#293]
I'm attempting to save image data from a CanvasRenderTarget to a file using the following code:
```
            FileSavePicker savePicker = new FileSavePicker();
            savePicker.SuggestedStartLocation = PickerLocationId.DocumentsLibrary;
            savePicker.FileTypeChoices.Add("PNG", new List<string> { ".png" });
            StorageFile file = await savePicker.PickSaveFileAsync();

            using (var fileStream = await file.OpenAsync(FileAccessMode.ReadWrite))
            {
                await RenderTarget.SaveAsync(fileStream, CanvasBitmapFileFormat.Png, 0.9f);
            }

```

I've attached an image from the way the content from the RenderTarget looks on the screen and how it looks from the png file after saving.  It looks good on screen, but it looks like the alpha is not translating to the saved file because all the opaque areas turn gray.  The CanvasRenderTarget is created with the default settings, but I've tried creating a new CanvasRenderTarget with different AlphaMode values but it seems to not work with anything other than Premultiplied.  I'm not even sure that is the problem.

Is this a shortcoming of the CanvasRenderTarget or is there something I can do to fix it?

![onscreen](https://cloud.githubusercontent.com/assets/7303539/13518811/20ec87f4-e18f-11e5-93fb-bf621d4d5862.PNG)
![saved](https://cloud.githubusercontent.com/assets/7303539/13518813/22cb98f8-e18f-11e5-8399-bee6caf744fb.PNG)

**Comment by shawnhar** [comment]
Can you explain more about how you created these two images please?  Is the second one a screen capture from some image viewer, or the result of loading the saved image back into Win2D?

**Comment by kellylawson** [comment]
Sure - both are screen captures.  The first is a screen capture from my app displaying the same CanvasRenderTarget content in a CanvasControl.  The second is a screen capture taken from the saved image being displayed in Chrome - not loaded back into Win2D.

**Comment by shawnhar** [comment]
That looks to me like Chrome is rendering premultiplied image data as unpremultiplied.  Have you checked the output image in different viewers?

It would help if you could share a repro app.

**Comment by kellylawson** [comment]
Yeah it shows that way in freshpaint and the windows photo viewer app as well.  I'll create a basic app to reproduce and send it over - I should be able to do it this weekend.

**Comment by kellylawson** [comment]
Just a quick update: I'm working on a test project but its functioning the way I would expect; saved canvas images have the correct colors.  This is looking like I have some side effects in my main project somewhere.  I'll update tomorrow after I work on it a bit more.

**Comment by kellylawson** [comment]
Ok, I finally got back to this and sorted it out.  Looks like the issue I was running into has already been fixed.  I created a new project with all the same code and it worked fine; upon investigating it seems my original project was still using an old version of Win2d (1.10), while the new project was using 1.15.  Updating the version fixes the issue.

Sorry to waste your time, I'll keep on the latest version from now on.
